packages/web-components/fast-element/src/templating/binding.spec.ts

```diff
@@ -189,7 +189,24 @@ describe("The HTML binding directive", () => {
 
             expect(toHTML(parentNode)).to.equal(`This is a template. value`);
         });
-    });
+
+        it("Should allow HTML tags in template bindings", async () => {
```

Comment:
I want to clarify this test-case description but I'm having trouble finding a good one. The expression in this test using the `x.knownValue` isn't a "binding" per-se, just an interpolation. I think what this really tests are that:
a.) Tags can be interpolated during template creation (you added a test specifically for this in template.spec.ts)
b.) Bindings that return templates are evaluated and re-rendered when observable properties used in the returned template are changed.

I actually don't see a test case for *b* in binding.spec.ts (maybe I'm missing or it is covered elsewhere?), so it might be worth generalizing this test-case instead, but mostly I want to avoid any confusion that FAST supports *binding* HTML tags in templates, because it only really supports interpolation.

Reply:
yea perhaps binding is the wrong word. Basically just something so that if I define a function on a FASTElement that returns an HTML tag, I want to be able to use that tag in my template. I'll change this to "Should allow interpolated HTML tags"